src/test/java/uk/gov/hmcts/reform/finrem/notifications/service/EmailServiceTest.java

```diff
@@ -280,4 +282,31 @@ public void shouldBuildTemplateVarsForConsented() {
         assertNull(returnedTemplateVars.get("courtName"));
         assertNull(returnedTemplateVars.get("courtEmail"));
     }
+
+    @Test
+    public void shouldBuildTemplateVarsForGeneralEmailContested() {
+        setContestedData();
+        notificationRequest.setGeneralEmailBody("test email body");
+
+        Map<String, String> returnedTemplateVars =
+                emailService.buildTemplateVars(notificationRequest, FR_CONTESTED_GENERAL_EMAIL.name());
+
+        assertNotNull(returnedTemplateVars.get("courtName"));
+        assertNotNull(returnedTemplateVars.get("courtEmail"));
+        assertNotNull(returnedTemplateVars.get("generalEmailBody"));
```

Comment:
Be better to be more verbose and  properly assert the result instead of just assertNotNull for these tests:
"assertEquals(returnedTemplateVars.get("generalEmailBody"), "test email body");"

Reply:
Good point, done